### PR TITLE
I added Travis Ci 

### PR DESCRIPTION
**Pull request by OussEmaDevCode**
In this pull request:
- in this pull request I add Travis CI to the project
- i added a badge to the read me file 
- please see the last commit

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+ - "3.6"
+install: pip install -r requirements.txt
+script:
+  - python manage.py
```
Comment (jatinkatyal13) [inline .travis.yml]: `python manage.py test`
It runs the Django tests that we have defined for our application
Also will handle the database part?

File: test.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+print("Hello world!")
+
```
Comment (jatinkatyal13) [inline test.py]: We don't need this file anymore!

**Comment by OussEmaDevCode** [comment]
I am trying to add that but I am having this error:

` raise UndefinedValueError('{} not found. Declare it as envvar or define a default value.'.format(option))
decouple.UndefinedValueError: SECRET_KEY not found. Declare it as envvar or define a default value.
The command "python manage.py" exited with 1.`

**Comment by OussEmaDevCode** [comment]
I tryed different configs but none of them worked there is an issue in that file!!!!!!!!!

**Comment by jatinkatyal13** [comment]
Actual there are a few environment variables that are needed to be set to run the application! You can find them in the .env.sample file!

**Comment by OussEmaDevCode** [comment]
And to what should I assign the DEBUG env

**Comment by jatinkatyal13** [comment]
Ideally True but then you'll need to add environment variables for postgres details else False

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # Certificate Generator Server
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/06bd6acff9cd4506985192596642ef5f)](https://www.codacy.com/app/JBossOutreach/certificate-generator-server?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=JBossOutreach/certificate-generator-server&amp;utm_campaign=Badge_Grade)
+[![Build Status](https://travis-ci.org/OussEmaDevCode/certificate-generator-server.svg?branch=master)](https://travis-ci.org/OussEmaDevCode/certificate-generator-server)
```
Comment (aashutoshrathi) [inline README.md]: We show Build Status of main repository's master branch.
So, please replace all `OussEmaDevCode` with `JBossOutreach` in Line 3

File: test.py [hunk 4]
```diff
@@ -0,0 +1,2 @@
+print("Hello world!")
```
Comment (aashutoshrathi) [inline test.py]: Yeah, actually even if you add tests, make sure you assert something.

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+ - "3.6"
+install:  
+ - pip install -r requirements.txt
+script:
+  - python test.py
+
```
Comment (aashutoshrathi) [inline .travis.yml]: Make Travis more like [this](https://github.com/aashutoshrathi/labper/blob/master/.travis.yml)
Add random values to all env variables we have for this repo. And luckily we are using the same config as that repo of mine, i.e. we are using Postgres for the server.
So, keep Postgres and it's script.

**Pull request closed without merging**